stylesheets/desktop/stylesheet-adoptions.css

```diff
@@ -0,0 +1,212 @@
+* {
+  margin: 0;
+  font-family: "Georgia", serif;
+}
+
+header {
+  background-color: #093C60;
+  padding: 5% 0% 5% 0%;
+  position: relative;
+}
+
+.header-intro {
+  margin-left: 10%;
+}
+
+.header-checkout {
+  position: absolute;
+  right: 0%;
+  top: 7%;
+}
+
+.header-logo {
+  width: 30%;
+}
+
+.header-title {
+  color: #81B741;
+  font-size: 1.5em;
+  width: 40%;
+}
+
+.cart-logo {
+  width: 10%;
+}
+
+#header-price {
+  color: white;
+  display: inline-block;
+}
+
+.button {
+  position:absolute;
+  right: 8%;
+  top: 20%;
+  background-color: #81B741;
+  border: none;
+  color: white;
+  padding: 1% 3%;
+  text-align: center;
+  text-decoration: none;
+  font-size: 1em;
+}
+
+.global-navigation {
+  display: flex;
+  font-size: 1em;
+  font-weight: bold;
+  background: #ECECEC;
+  padding: 2% 0% 2% 0%;
+}
+
+.nav-link {
+  color: #093C60;
+  text-decoration: none;
+  padding: 0% 1% 0 15%;
+}
+
+.nav-link-2 {
+  color: #093C60;
+  text-decoration: none;
+  padding: 0% 1% 0 1%;
+}
+
+.grid-index-adoptions {
+  display: grid;
+  grid-template-columns: 20% 20% 20% 20%;
+  grid-gap: 3%;
+  margin: 3% 0% 10% 15%;
+  line-height: 1.5em;
+}
+
+.adoption-image {
+  max-width: 100%;
+}
+
+.adoption-header {
+  color: #093C60;
+  padding: 1% 0% 3% 3%;
+}
+
+.adoption-cost {
+  color: #3D3D3D;
+  padding: 0% 0% 3% 3%;
+}
+
+.adoption-body {
+  color: #3D3D3D;
+  font-size: 1em;
+  padding: 0% 5% 3% 3%;
+}
+
+.index-adopt-button {
+  margin: 2% 4%;
+  background-color: #81B741;
+  border: none;
+  color: white;
+  padding: 3% 30%;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 1em;
+  font-weight: bolder;
+}
+
+#dogs-header {
+  color: #093C60;
+  margin: 2% 0% 0% 15%;
+}
+
+#murphy {
+  grid-column: 1;
+  grid-row: 1;
+  border: 1px solid #ECECEC;
+}
+
+#poppy {
+  grid-column: 2;
+  grid-row: 1;
+  border: 1px solid #ECECEC;
+}
+
+#jack {
+  grid-column: 3;
+  grid-row: 1;
+  border: 1px solid #ECECEC;
```

Comment:
One of the few places I think its ok to use px. but still be careful of how it looks at different device resolutions.